.changeset/little-avocados-fetch.md

```diff
@@ -0,0 +1,9 @@
+---
+"@remix-run/css-bundle": patch
+"@remix-run/dev": patch
```

Comment:
I believe we should bump to `minor` for this one since it's a new feature.

Reply:
Fixed.